tests/auth_tests/test_validators.py

```diff
@@ -145,6 +145,24 @@ def test_help_text(self):
         )
 
 
+class MinimumLengthValidatorCustomErrorTest(SimpleTestCase):
+    def test(self):
+        class CustomMinimumLengthValidator(MinimumLengthValidator):
+            def get_error_message(self):
+                return "Your password must be %d characters long" % self.min_length
+
+        expected_error = "Your password must be %d characters long"
+
+        with self.assertRaises(ValidationError) as cm:
+            CustomMinimumLengthValidator().validate("1234567")
+        self.assertEqual(cm.exception.messages, [expected_error % 8])
```

Comment:
We can use `assertRaisesMessage` in the tests
```suggestion
        expected_error = "Your password must be 8 characters long"

        with self.assertRaisesMessage(ValidationError, expected_error) as cm:
            CustomMinimumLengthValidator().validate("1234567")
```